tfhe/src/core_crypto/algorithms/test/ggsw_encryption.rs

```diff
@@ -72,7 +72,7 @@ fn test_parallel_and_seeded_ggsw_encryption_equivalence<Scalar>(
         );
 
         let mut par_ggsw = GgswCiphertext::new(
-            Scalar::ZERO,
+            Scalar::MAX,
```

Comment:
Why change these values?
Is there a specific reason to take MAX in particular?

Reply:
hmm actually it might not be enough to trigger the bug the hw encountered :/

Reply:
I just wanted some broken value and did not want to run the rng as it would slow the coverage down potentially